src/pages/page404.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { ErrorPage } from "../components/ErrorBoundary/ErrorPage";
+
+export const Page404 = () => {
+  return <ErrorPage title="404" description="Oops, page not found" />;
```

Comment:
```suggestion
export const PageNotFound = () => {
  return <ErrorPage title="404" description="Oops, page not found" />;
```

minor nitpick on naming to more descriptive

Reply:
ok